package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "babel src --out-dir lib",
     "watch": "npm run babel -- --watch",
-    "examples": "webpack-dev-server",
+    "examples:babel": "webpack-dev-server --config webpack.config.babel.js",
+    "examples:rollup": "webpack-dev-server --config webpack.config.rollup.js",
```

Comment:
what is this for?

Reply:
The two npm scripts? One for babel and other for rollup. Did you mean to ask something else?

Reply:
i'm confused why the rollup build would run through webpack, the should be mutually exclusive tools